draft-ietf-quic-transport.md

```diff
@@ -2089,15 +2091,18 @@ different local addresses, as discussed in {{connection-id}}.  For this to be
 effective endpoints need to ensure that connections IDs they provide cannot be
 linked by any other entity.
 
-This eliminates the use of the connection ID for linking activity from
-the same connection on different networks.  Header protection ensures
-that packet numbers cannot be used to correlate activity.  This does not prevent
-other properties of packets, such as timing and size, from being used to
+Endpoints MAY move to a new connection ID at any time.
```

Comment:
Nit: move to may not be clear.  How about "Endpoints MAY change the Destination Connection ID they send in short header packets at any time." assuming that statement is correct.